README.md

```diff
@@ -23,6 +23,18 @@
 
 The DEV API does not have a way of uploading images yet. If we submit a Markdown content with relative paths of image links, DEV will not be able to show those images. As a workaround of this problem, we need to provide a full path for the images either manually via the `devto.yml` file or using the `--prefix` flag.
 
+The Markdown file must contains at least the title property of the Jekyll front matter, like in:
+
+```
+---
+title: An example title
+published: true
```

Comment:
```suggestion
```
I would like to avoid including `published` field in here. The reason is that `devto submit` has a `--published` flag and `published` field in front matter will override the flag due to the nature of the [API](https://docs.dev.to/api/#operation/createArticle).

> If the markdown contains a front matter, it will take precedence on the equivalent params given in the payload.

I admit there can be improvement in the UX for `submit` subcommand but for now the best might be not including it in the doc because user might copy this snippet and use it and find that their article is published unexpectly.

Reply:
Totally agree